src/Compilers/CSharp/Test/Emit/Emit/EditAndContinue/EditAndContinueTests.cs

```diff
@@ -10504,79 +10501,103 @@ .maxstack  1
         }
 
         [Fact]
-        public void AddUsing_AmbiguousCode()
+        public void Records_AddWellKnownMember()
         {
-            var source0 = MarkedSource(@"
-using System.Threading;
-
-class C
+            var source0 =
+@"
+namespace N
 {
-    static void E() 
+    record R(int X)
     {
-        var t = new Timer(s => System.Console.WriteLine(s));
     }
-}");
-            var source1 = MarkedSource(@"
-using System.Threading;
-using System.Timers;
-
-class C
+}
+";
+            var source1 =
+@"
+namespace N
 {
-    static void E() 
-    {
-        var t = new Timer(s => System.Console.WriteLine(s));
-    }
-    
-    static void G() 
+    record R(int X)
     {
-        System.Console.WriteLine(new TimersDescriptionAttribute(""""));
+        protected virtual bool PrintMembers(System.Text.StringBuilder builder)
+        {
+            return true;
+        }
     }
-}");
-
-            var compilation0 = CreateCompilation(source0.Tree, targetFramework: TargetFramework.NetStandard20, options: ComSafeDebugDll);
-            var compilation1 = compilation0.WithSource(source1.Tree);
-
-            var e0 = compilation0.GetMember<MethodSymbol>("C.E");
-            var e1 = compilation1.GetMember<MethodSymbol>("C.E");
-            var g1 = compilation1.GetMember<MethodSymbol>("C.G");
```

Comment:
>} [](start = 8, length = 1)

You might want to add additional validation - just checking synthesized members does not do much (since records don't actually produce synthesized members).

It'd be useful to validate the definition names emitted and/or EncLog table - see `ModifyMethod` test.